src/components/StyleGuide/CardsAsNavigation.mdx

```diff
@@ -0,0 +1,199 @@
+import { Canvas, Meta, Source, Unstyled } from "@storybook/addon-docs/blocks";
+
+import Banner from "../Banner/Banner";
+import Card, { CardActions, CardContent, CardHeading } from "../Card/Card";
+import Link from "../Link/Link";
+import SimpleGrid from "../Grid/SimpleGrid";
+import ComponentDocsHeader from "../../utils/components/ComponentDocsHeader";
+import ImageRevamp from "../../utils/ImageExample";
+import { getPlaceholderImage } from "../../utils/utils";
+
+import { distinctLinksCard, fullClickCard } from "./CardsAsNavigationData";
+
+<Meta title="Development Guide/Cards as Navigation Links" />
+
+<ComponentDocsHeader
+  category="Style Guide"
+  componentName="Cards as Navigation Links"
+  summary="Guidelines for for using cards as navigation elements"
+/>
+
+## Table of contents
+
+- [General information](#general-information)
+- [Full-click functionality](#full-click-functionality)
+- [Distinct links](#distinct-links)
+
+## General information
+
+Cards are used extensively throughout NYPL web apps for navigational purposes.
+They are used in search results and on landing pages. They can represent
+distinct items, like books and events, or signify site landing pages and
+sections. They may contain a single link or present multiple links and buttons.
+
+## Full-click functionality
+
+Navigational cards designed to link to a singular destination should be
+configured with [full-click
+functionality](/docs/components-basic-elements-card--docs#card-with-full-click-functionality),
```

Comment:
I think this is a known issue, but this doesn't actually bring the user to right section.

Reply:
Yeah, this is something that I also noticed. It's as if that section of the page is not yet loaded when the browser tries to scroll down to it. And so it just gets close.